indico/modules/events/registration/models/items.py

```diff
@@ -97,6 +132,22 @@ class RegistrationFormSection(RegistrationFormItem):
         'polymorphic_identity': RegistrationFormItemType.section
     }
 
+    @property
+    def locator(self):
+        return dict(self.registration_form.locator, section_id=self.id)
+
+    @property
+    def angular_data(self):
+        return {
+            'id': self.id,
+            'title': self.title,
+            'description': self.description,
+            'items': [child.angular_data for child in self.children if not child.is_deleted],
+            'enabled': self.enabled,
+            '_type': 'GeneralSectionForm',
```

Comment:
does the angular code use `_type`? if not let's remove it. otherwise let's use decent types instead of the legacy class names